doc/help/kdb-rm.md

```diff
@@ -11,6 +11,10 @@ Note that when using the `-r` flag, not only the key directly at `path` will be
 
 This command removes key(s) from the Key database.
 
+## LIMITATIONS
+
+Cascading keys (i.e. keys that start with `/`) are only allowed using the `-f` flag.
```

Comment:
Why? And what is the behavior? I would expect it removes any key in any namespace. What is ambiguous here?

I would also avoid giving -f so different meanings: bypassing spec: validation (which also makes sense for kdb-rm) and allowing cascading writes. What if someone does not want to bypass validation but would like to use cascading?

Reply:
My rationale was to implement https://github.com/ElektraInitiative/libelektra/issues/3742 (I interpreted this to mean that all cascading operations on `kdb rm` were meant to be disabled, as not otherwise specified).
The -f option was intended to keep the old behavior available, but of course this is not an optimal solution.
I will revert it back.

Reply:
Yes, I think in non-ambiguous cases we can simply remove the key(s). `-f` should be only for disabling the spec validation.

The remaining problem is that repeated executions of `kdb rm -r /something` might remove several keys. Please think about it if we really want the same semantics as used in `kdb set`. How is the behavior in FUSE when you remove a file that is "backed up" by several keys (in several namespaces)?

Reply:
The old behavior is back.

Although it is not an advisable strategy to do that, I think it is better to not introduce a new special case here. and keep things consistent with the other commands.

In the FUSE tool this case does not occur, since the cascading namespace is mounted read-only (This was decided because a file system that only supports writing partially (due to the ambiguity of creating keys in the cascading namespace) is a mess regarding compatibility with tools and usability for users).